spec/lib/email/receiver_spec.rb

```diff
@@ -756,6 +756,16 @@ def create_post_reply_key(value)
       MD
     end
 
+    it "tries not to repeat duplicate attachments" do
+      SiteSetting.authorized_extensions = "jpg"
+
+      expect { process(:logo_1) }.to change { UploadReference.count }.by(1)
```

Comment:
`logo_1` is the first email that has 1 attachment (the discourse logo) so it should generate one `Upload` record and one `UploadReference` record.